tutorial/README.rst

```diff
@@ -13,5 +13,11 @@ The order we assume is as follows:
 * :ref:`pruning`
 * :ref:`sampler`
 
+**Other Resources**:
+
+* `Examples`_: Demonstarate how to use Optuna with popular libraries for machine learning and deep learning.
```

Comment:
A details, but those examples are not limited to integrations (e.g. the quadratic function example). How about something along "For more examples including how to use Optuna with... "?